package.json

```diff
@@ -151,6 +151,13 @@
     "vscode-nls": "^2.0.2",
     "zone.js": "^0.6.26"
   },
+  "resolutions": {
+    "pm-mocha-jenkins-reporter/mocha/minimatch": "^3.0.5",
```

Comment:
So there are no runtime dependencies using minimatch? If that's the case we can just skip the update for now - resolutions isn't something we should be doing unless we absolutely need to.

Reply:
`ejs` is the only runtime dep using minimatch, all other ones are dev dep. And we only use `ejs` for rendering (in one line of code), no feature related to regex is used.

Reply:
Alright. If you aren't able to update the root packages for these then just revert the changes and we can dismiss the alert.